src/utils/links.ts

```diff
@@ -15,16 +15,22 @@ export function openGitifyReleaseNotes(version: string) {
   );
 }
 
-export function openGitHubNotifications() {
-  openExternalLink('https://github.com/notifications' as Link);
+export function openGitHubNotifications(account: Account) {
```

Comment:
Maybe a good idea to default the hostname to `github.com`, I'm not sure if the sidebar is available while no accounts are present.

For simplicity of use, the link handlers can be grouped into a custom hook and pull needed context on their own.

Reply:
Great feedback: updated to handle the unauthenticated scenario